src/odemis/gui/model/__init__.py

```diff
@@ -1293,6 +1306,10 @@ def __init__(self, main):
 
         self.selected_scintillators = model.ListVA([])  # set of ints, overview images to be acquired
 
+        self.is_calib = model.BooleanVA(False)
+        # indicates whether the microscope is calibrating
+        self.is_calibrating = model.BooleanVA(False)
```

Comment:
Note that there is some code in the GUI that looks at `is_acquiring` and prevents the user from closing the window if it's True (and tells the user that it should first stop the acquisition). Does the same thing need to happen if it's calibrating? (I'd tend to answer "no", because it's fine to close the GUI, in which case the calibration will eventually end, and next time the GUI is started, the calibration will have to be redone. Not a big deal as loosing acquisition data).

Reply:
Good point. The calibrations are indeed themselves independent from the front-end. I am wondering about the part, when the calibration would return to the calibration manager and store the calibration values (which is ok to loose) and put the system back into a good state (which is for some calibrations important). So if the GUI is closed during a calibration that modifies the scan amplitude and usually at the end would put back the original value, that would not happen anymore right? On the other hand we indeed then run the calibrations again and those don't depend on VA settings but do use factory or calibrated values. So most likely it is not a problem indeed.